Dolphin/windWakerInterface.py

```diff
@@ -159,7 +163,7 @@ def downgrade_wallet():
 
 def give_rupees(amount: int):
     if random_rupoors and bool(random.getrandbits(1)):
-            amount = ((amount ^ 0xFFFFFFFF) + 1 )
+            amount = -amount + 2**32
```

Comment:
I'm still not entirely sure why this fix was required. I've tested out this function before the last minor release (0.0.2.2 on February 27), and I haven't heard anything about it. Do you have an example?

Reply:
My changes were actually not needed. The initial method you wrote works fine. 
I was still on the mindset that it wasn't working as I hadn't heard anything from you about that.

Reply:
No worries. If you could remove the changes then that would be great.
The current solution makes the 2's Complement of the Number, so it's mirroring the Hardware's Expectations.
Your solution likely does the same, it's just not as clear.